### ISSUE TITLE
tiny issue with the print out of a warning in   hemodynamic_models.py

### ISSUE THREAD

**Issue by chrplr** [nistats/nistats#176]
Here is how the warning is displayed in the terminal during execution:

.../python3.6/site-packages/nistats-0.0.1a0-py3.6.egg/nistats/hemodynamic_models.py:275: UserWarning: Some stimulus onsets are earlier than %d in the experiment and are thus not considered in the model  % (frame_times[0] + min_onset)), UserWarning)

The '%d' string formatting operator is not applied. 

The lines in the sources:

```python
    if (onsets < frame_times[0] + min_onset).any():
        warnings.warn(('Some stimulus onsets are earlier than %d in the' +
                       ' experiment and are thus not considered in the model'
                % (frame_times[0] + min_onset)), UserWarning)
```

**Comment by bthirion** [comment]
oops.

**Comment by kchawla-pi** [comment]
Fixed in #208
